### PR TITLE
Add translation for flash messages

### PR DESCRIPTION
**Pull request by albiere**
It adds translations for "Logged In" in English and Spanish.

Could you guys have a look at it? @sbfaulkner @mattvleming

### CODE REVIEW

**Comment by mattvleming** [comment]
Beautiful. The version number should probs be updated. Also just checking, do we have any more translations to add?

**Comment by sbfaulkner** [comment]
LGTM... but, we should probabaly get the other strings in the concern translated too

**Comment by albiere** [comment]
Just added some extra keys to our locales files.

File: config/locales/es.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+es:
+  logged_in: 'Has iniciado sesión'
+  logged_out: 'Cerrado sesión'
```
Comment (julionc) [inline config/locales/es.yml]: Spanish native speaker here. I'd recommend to edit the following strings to:
* logged_out: `Su sesión ha finalizado correctamente` or `Cerrando sesión` (Although it sounds a bit weird)
* could_not_log_in: `No fue posible iniciar sesión en la tienda`

**Comment by hcoleman** [comment]
@julionc 

How about:
could_not_log_in: "No se pudo iniciar sesión en tu tienda de Shopify"
logged_out: Cerrar sesión

cc: @albiere

**Comment by kevinhughes27** [comment]
Code looks good to me and thanks for adding more tests - curious why the translation though since Shopify itself is not translated.

**Comment by mattvleming** [comment]
We have a Rails app that uses shopify_app and is being translated for Spanish users. Feel free to DM Angelo or myself if you want to know which one (since this discussion is public).

**Comment by kevinhughes27** [comment]
Okay awesome! That works for me I just wasn't sure if this was being done for a reason or not. :shipit:

**Comment by mattvleming** [comment]
LGTM

**Pull request merged**